CHANGES.rst

```diff
@@ -11,6 +11,9 @@ New Features
 
 - User-friendly API access to plugins. [#1401]
 
+- Line Lists show which medium the catalog wavelengths were measured in,
+  in accordance to the metadata entry [#1626]
```

Comment:
```suggestion
- Line Lists show whether the catalog wavelengths were measured in air or vacuum. [#1626]
```

I'm going off of @PatrickOgle's statement that "medium" without context might not be informative.

Reply:
Although now that I read it again I guess the context is there 😆